control-plane/subcommand/server-acl-init/command_ent_test.go

```diff
@@ -321,6 +322,11 @@ func TestRun_ACLPolicyUpdates(t *testing.T) {
 			require.NoError(t, err)
 
 			// Check that the expected policies were created.
+			// There will be more policies returned in the List API that are defaults
+			// existing in Consul on startup, including but not limited to:
+			// * global-management
+			// * builtin/global-read-only
+			// * agent-token
```

Comment:
We added a default policy and the counting broke. I don't see the point in testing the count - we already test that the policies we expect to create exist below. The fact that there are defaults is extraneous and brittle.